apps/next-app/src/pages/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react"
+import Image from 'next/image'
+import { Button } from 'ui/Button'
```

Comment:
Separate app/package imports after absolute imports and pad them with a new line.

Reply:
done